spec/helpers.lua

```diff
@@ -7,7 +7,7 @@
 
 local BIN_PATH = "bin/kong"
 local TEST_CONF_PATH = os.getenv("KONG_SPEC_TEST_CONF_PATH") or "spec/kong_tests.conf"
-local CUSTOM_PLUGIN_PATH = "./spec/fixtures/custom_plugins/?.lua"
+local CUSTOM_PLUGIN_PATH = os.getenv("KONG_LUA_PACKAGE_PATH") or "./spec/fixtures/custom_plugins/?.lua"
```

Comment:
Regular tests will fail if `spec/fixtures/custom_plugins` is not there, right? Maybe this should always concatenate that to the path?

Reply:
yes, I have updated PR  @hishamhm